app/templates/prepare_db_for_dev.example.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Returns true if module is enabled, false otherwise.
+function module_is_enable() {
+  local MODULE_IS_ENABLED="`drush pm-list --pipe --type=module --status=enabled --no-core | grep $1`"
+  if [ -z "$MODULE_IS_ENABLED" ]; then
+    # Module is disabled.
+    return 1
+  else
+    # Module is enabled.
+    return 0
+  fi
+}
+
+# Define the root of the GIT repository.
+cd ${0%/*}
+ROOT=$(pwd)
+cd $ROOT
+
+# Load the colors.
+source $ROOT/scripts/helper-colors.sh
+
+echo
+echo -e "${BLUE} > You're about to modify your database.${RESTORE}"
+read -p "Are you sure (y/n)? "
+
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    exit
+fi
+
+echo
+echo -e "${LBLUE} > Preparing your database for development.${RESTORE}"
+
+cd $ROOT/www
+
+if module_is_enable "search_api"; then
+  # If this module was never enabled on the current project then the
+  # "search_api_index" table was never exists and we get the message
+  # "Query failed".
+  echo
+  echo -e "${LBLUE} > Changing the solr index to be read only.${RESTORE}"
+  drush sqlq "UPDATE search_api_index SET read_only = '1'"
+fi
+
+echo
```

Comment:
Handling Pantheon Apache Solr settings
```SH
echo
echo -e "${LBLUE} > Removing Pantheon Apache Solr settings.${RESTORE}"
# Disabling pantheon modules.
drush sqlq "UPDATE system SET status = 0 WHERE name LIKE '%pantheon%'"
# Removing this variable prevent a fatal error since Pantheon set this variable
# but we don't have this class locally. 
drush vdel apachesolr_service_class --yes
```